sway/tree/container.c

```diff
@@ -420,7 +420,12 @@ void container_arrange_title_bar(struct sway_container *con) {
 	wlr_scene_rect_set_clipped_region(con->title_bar.background, (struct clipped_region) {
 			.corner_radius = 0,
 			.corners = CORNER_LOCATION_NONE,
-			.area = text_box,
+			.area = {
+				.x = text_box.x,
+				.y = text_box.y,
+				.width = text_box.width - thickness,
```

Comment:
I don't see how the thickness would impact the size of the title texture, and didn't see any issue in my (relatively extensive) testing, how did you come across this requirement?

Reply:
It's affected by the `titlebar_border_thickness` config option. Here's without the compensation:

![image](https://github.com/user-attachments/assets/30626f97-e271-44cf-ab91-56124800a590)

Reply:
Probably because the text node is in the same tree as the background rect (which also has the same offset)? That's my guess at least 😅

Reply:
Makes sense, I suspect it's more positional than size then. I'll try it with opacity this evening

Reply:
looks like my theory may have been correct. Fixed! This had issues with opacity due to pos overlap